Interfaces/API/Dirac.py

```diff
@@ -938,7 +941,7 @@ def _getLocalInputData(self, parameters):
     if inputData:
       if isinstance( inputData, basestring ):
         inputData = [inputData]
-    return inputData
+    return S_OK(inputData)
```

Comment:
Why ? `_getLocalInputData` is protected.

Reply:
...and?

Reply:
and I thought we accept that protected/private method do not return necessarily S_OK/S_ERROR, especially that this can only be S_OK

Reply:
It's a good point, and maybe I should un-protect it because I have actually overridden this specific method in LHCbDIRAC (and there I definitely need to return S_OK/S_ERROR). Would it be better?

Reply:
Since we do not have a strong policy on that, I don't have a strong opinion on this particular case. I was just surprised of that change that seemed to come for no reason. But I am happy if it stays protected if it is not meant to be used by API users.

Reply:
OK then I leave it as it is.